Doing ifconfig athX down/ifconfig athX up several times provokes kernel crash.
See http://madwifi.org/ticket/914. But if ath_hal_phydisable() is skipped  -
I observe NO CRASH whatsoever ... :\ Weird.

Signed-off-by: Mindaugas Kriaučiūnas <mindaugas.kriauciunas@gmail.com>
Signed-off-by: Žilvinas Valinskas <valins@soften.ktu.lt>

Index: madwifi-ng-trunk/ath/if_ath.c
===================================================================
--- madwifi-ng-trunk.orig/ath/if_ath.c	2006-10-16 17:40:50.000000000 +0300
+++ madwifi-ng-trunk/ath/if_ath.c	2006-10-18 16:17:32.000000000 +0300
@@ -1997,7 +1997,10 @@
 		ath_draintxq(sc);
 		if (!sc->sc_invalid) {
 			ath_stoprecv(sc);
+
+			/* XXX: this helps to avoid crashes on ifconfig down/up
 			ath_hal_phydisable(ah);
+			 */
 		} else
 			sc->sc_rxlink = NULL;
 		ath_beacon_free(sc);		/* XXX needed? */
